packages/common/lib/types/Generic.types.ts

```diff
@@ -190,25 +240,25 @@ export interface CredentialRequestSdJwtVc extends CommonCredentialRequest {
 }
 
 export interface CommonCredentialResponse {
-  format: string;
+  // format: string;  TODO do we still need this for previous version support?
```

Comment:
Yeah, this was used for matching credential types previously. Not sure how tests can still pass if this is gone. Or has it been introduced elsewhere? Same for the other `format` properties that have been commented in this file